packages/shared/labels/labels.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { Label, LabelGroup } from '@patternfly/react-core';
+
+export const Labels: React.FC<LabelsProps> = ({
+  labels,
+  collapsedText,
+  expandedText,
+  numLabels,
+  className,
+}) => {
+  return (
+    <LabelGroup
+      numLabels={numLabels}
+      expandedText={expandedText}
+      collapsedText={collapsedText}
+    >
+      {labels.map((label) => (
+        <Label key={label} className={className} isTruncated>
+          {label}
+        </Label>
+      ))}
+    </LabelGroup>
+  );
+};
+
+export type LabelsProps = {
```

Comment:
there should be something defined for `LabelGroup`, most probably be called `LabelGroup`.
```suggestion
export type LabelsProps = LabelGroupProps & {
```